File: Examen Parcial/20221309_EX1/Pregunta03/TransitSoft/TransitSoftModel/PropietarioDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransitSoftModel
{
    public class PropietarioDTO
    {
        private int? id;
        private string dni;
        private string nombres;
        private string apellidos;
        private string direccion;

        public PropietarioDTO()
        {
            this.Id = null;
            this.Dni = null;
            this.Nombres = null;
            this.Apellidos = null;
            this.Direccion = null;
        }

        public PropietarioDTO(int? id, string dni, string nombres, string apellidos, string direccion)
        {
            this.Id = id;
            this.Dni = dni;
            this.Nombres = nombres;
            this.Apellidos = apellidos;
            this.Direccion = direccion;
        }

        public int? Id { get => id; set => id = value; }
        public string Dni { get => dni; set => dni = value; }
        public string Nombres { get => nombres; set => nombres = value; }
        public string Apellidos { get => apellidos; set => apellidos = value; }
        public string Direccion { get => direccion; set => direccion = value; }
    }
}

File: EX2/Conexion SOAP Transitsoft/TransitSoftModel/src/main/java/pe/edu/pucp/transitsoft/model/PropietarioDTO.java
package pe.edu.pucp.transitsoft.model;

public class PropietarioDTO {
    private Integer id;
    private String dni;
    private String nombres;
    private String apellidos;
    private String direccion;

    public PropietarioDTO() {
        this.id = null;
        this.dni = null;
        this.nombres = null;
        this.apellidos = null;
        this.direccion = null;
    }

    public PropietarioDTO(Integer id, Integer dni, String nombres, String apellidos, String direccion) {
        this.id = id;
        this.dni = dni;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.direccion = direccion;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getDni() {
        return dni;
    }

    public void setDni(String dni) {
        this.dni = dni;
    }

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }
}
